src/main/java/io/getunleash/repository/SegmentCollection.java

```diff
@@ -0,0 +1,44 @@
+package io.getunleash.repository;
+
+import io.getunleash.Segment;
+import io.getunleash.lang.Nullable;
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public final class SegmentCollection implements Serializable {
```

Comment:
Why is this serializable?

Reply:
/cc @andreas-unleash

Reply:
I assume it needs to be in order to be able to write it to the file backup. (or gson supports anything..)